qa/suites/fs/shell/distro

```diff
@@ -1 +1 @@
-.qa/distros/supported-random-distro$
\ No newline at end of file
+.qa/distros/supported
```

Comment:
`supported-random-distro$` and `supported` both point the same yamls, so is this change really needed?

Reply:
`supported-random-distro$` picks a random yaml in the directory. We want a matrix with all of the supported distros.